wpe/src/main/java/com/wpe/wpe/services/WPEServiceConnection.java

```diff
@@ -43,12 +43,17 @@ public final class WPEServiceConnection implements ServiceConnection {
     private final ProcessType processType;
     private ParcelFileDescriptor parcelFd;
 
-    public WPEServiceConnection(long pid, @NonNull ProcessType processType, @NonNull ParcelFileDescriptor parcelFd) {
+    private WPEServiceConnectionDelegate delegate;
```

Comment:
Should be `final` according to the usage you are doing of the reference later on.

Reply:
fixed accordingly